demo-terraform/aws/wordpress_ec2_alb_rds/security-groups.tf

```diff
@@ -1,38 +1,34 @@
 resource "aws_security_group" "web_server" {
-    name_prefix = "web-${random_string.lab_id.result}"
-    vpc_id      = module.vpc.vpc_id
-
-    ingress {
-        from_port = 80
-        to_port   = 80
-        protocol  = "tcp"
-
-        cidr_blocks = [
-          "10.0.0.0/8",
-        ]
-    }
-    ingress {
-        from_port = 22
-        to_port   = 22
-        protocol  = "tcp"
-
-        cidr_blocks = [
-          "0.0.0.0/0",
-        ]
-    }
-    egress {
-        from_port = 0
-        to_port   = 0
-        protocol  = "-1"
-        cidr_blocks = [
-          "0.0.0.0/0",
-        ]
-    }
+  name_prefix = "web-${random_string.lab_id.result}"
+  vpc_id      = module.vpc.vpc_id
+
+  ingress {
+    from_port = 80
+    to_port   = 80
+    protocol  = "tcp"
+
+    cidr_blocks = ["<cidr>"]
+  }
+  ingress {
+    from_port = 22
+    to_port   = 22
+    protocol  = "tcp"
+
+    cidr_blocks = ["<cidr>"]
+  }
+  egress {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure Security Groups do not have unrestricted specific ports open - (SSH,22)**
 
**Vulnerability**
Security Groups - Unrestricted Specific Ports - SSH,22. Exposed ports may allow attackers to access the application listening on that port.
 
**Remediation**
In AWS Console - 
 1. Sign in the AWS Console and go to the Amazon VPC console.
 2. In the navigation pane, select Security Groups.
 3. For each security group, perform the following actions:
      a) Select the security group.
      b) Select the Inbound Rules.
      c) Identify the rules to be removed.
      d) Remove the required rules.
 4. Click Save.

source: accurics-